### PR TITLE
Implement docsrv Makefile

### PR DESCRIPTION
**Pull request by dpordomingo**
This PR implements docsrv in docsrv so you can serve docsrv's documentation using docsrv.

blocks https://github.com/src-d/ci/pull/25

### CODE REVIEW

**Comment by Serabe** [comment]
1qdtcp

**Review by Serabe — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+		fi; \
+		echo "Installing 'ci'..."; \
+		cp $(CI_PATH)/$(MAKEFILE) .; \
+	fi; \
+
+-include $(MAKEFILE)
+
+#tools
+mkdir := mkdir -p
+goBuild := CGO_ENABLED=0 go build -o
```
Comment (erizocosmico) [inline Makefile]: s/goBuild/go_build/

**Review by erizocosmico — CHANGES_REQUESTED** [review]
Are we going to add docsrv'd docs to this project? Is it even going to be displayed in the landing?

**Comment by dpordomingo** [comment]
@erizocosmico I amended the PR unique commit with your suggestion  `s/goBuild/go_build/`
Answering your question about if this project will implement `docsrv` requirements, I'd say that all src-d public repositories should implement it (as said in the meeting for "defining our documentation")

**Comment by erizocosmico** [comment]
But if they're not going to be displayed in source{d}'s landing what's the point of documenting the projects in a way that they can be?

File: Makefile [hunk 2]
```diff
@@ -1,3 +1,32 @@
...
+
+#tools
+mkdir := mkdir -p
+go_build := CGO_ENABLED=0 go build -o
+
 build:
-	@mkdir -p bin; \
-	CGO_ENABLED=0 go build -o bin/docsrv docsrv.go
+	@$(mkdir) bin;
+	@$(go_build) bin/docsrv docsrv.go
```
Comment (smola) [inline Makefile]: Maybe these should go without `@` so the command is outputted to the terminal?

Comment (dpordomingo) [reply Makefile]: I'll do it, thanks

**Comment by dpordomingo** [comment]
@smola I removed the `@` from the `@$(go_build) bin/docsrv docsrv.go` command (and amended that change over the prev commit)
@erizocosmico it seems that all projects will appear in the landing

**Comment by erizocosmico** [comment]
You need to rebase

**Comment by dpordomingo** [comment]
@erizocosmico done

**Pull request merged**